### DISCUSSION TITLE
Why with some models, when I enter any text, I get a blank response?

### DISCUSSION THREAD

**Discussion by Nord1cWarr1or** [janhq/jan#1651] [Get Help]
For example, Yarn Mistral 7B Q4.

![изображение](https://github.com/janhq/jan/assets/47604048/b3af0e5c-6b48-4cf8-baf3-52c1f9c788e1)
![изображение](https://github.com/janhq/jan/assets/47604048/28cf66bb-a4be-4fc8-9343-85ab79d8f106)

I'm using AppImage build, on Arch-based distro. Jan version is 0.4.4-157

**Comment by LMcontent** [comment]
I got have this bug after install Cuda for work with GPU. But update, new install app and new download model can't  help me - Jan not working. Win 11 64, 16 Ram, Nvidia 1050Ti, new cuda from Nvidia site, Jan last version (stable or/and night). Jan don't work on CPU and don't work on GPU now.  Maybe I should uninstall cuda?

**Reply by LMcontent** [reply]
I uninstalled CUDA, models and reinstalled Jan and still received nothing in response.  But then I realized the reason, I haven't updated my video card drivers for almost 2 years.  I updated the video card drivers via GeForce Game Ready, and then reinstalled CUDA, Jan and downloaded the models.  This worked for me.  Jan is working.
